### PR TITLE
avoid ax-10--12 in elabg (and sbc6g)

### LINKED ISSUE #4257: elabg without ax-10 -- 12
assigned = icecream17

```
  ${
    $d A x y $.  $d ph y $.
    $( Lemma $)
    elab6g $p |- ( A e. V -> ( A e. { x | ph } <-> A. x ( x = A -> ph ) ) ) $=
      ( vy cv cab wcel weq wal wceq eleq1 eqeq2 imbi1d albidv wsb df-clab bitri
      wi sb6 vtoclbg ) EFZABGZHZBEIZASZBJZCUCHBFZCKZASZBJECDUBCUCLUBCKZUFUJBUKU
      EUIAUBCUHMNOUDABEPUGAEBQABETRUA $.
  $}

  ${
    $d ps x $.  $d A x $.
    elabg.1 $e |- ( x = A -> ( ph <-> ps ) ) $.
    elabg $p |- ( A e. V -> ( A e. { x | ph } <-> ps ) ) $=
      ( wcel cab cv wceq wi wal elab6g wex pm5.74i albii 19.23v bitri elisset
      wb pm5.5 syl syl5bb bitrd ) DEGZDACHGCIDJZAKZCLZBACDEMUHUFCNZBKZUEBUHUFBK
      ZCLUJUGUKCUFABFOPUFBCQRUEUIUJBTCDESUIBUAUBUCUD $
  $}

  ${
    $d A x $.
    $( Bonus (it was actually the original focus) $)
    sbc6g $p |- ( A e. V -> ( [. A / x ]. ph <-> A. x ( x = A -> ph ) ) ) $=
      ( wsbc cab wcel cv wceq wi wal df-sbc elab6g syl5bb ) ABCECABFGCDGBHCIAJB
      KABCLABCDMN $.
  $}
```

This obsoletes elabgw, elab2gw, and elrab2w

### PR DESCRIPTION
**Pull request by icecream17**
Fixes #4257

Commit by commit; #4264 saved several hundred which means the axiom counts in this pr are outdated

### CODE REVIEW

**Comment by benjub** [comment]
Out of curiosity, do you have the diff of the axiom usage file ? (not the total numbers you gave here, but axiom usage of each theorem, gotten with metamath-knife)

**Review by benjub — APPROVED** [review]

**Comment by icecream17** [comment]
I uploaded the diff at https://github.com/icecream17/Stuff/blob/main/math/~ax.diff

**Edit**: Looking at it, it seems like basically most of the 80 usages of ~ elab2g make a few downstream effects

File: set.mm [hunk 1]
```diff
@@ -52971,21 +52950,19 @@ of the restricted converse (see ~ cnvrescnv ).  (Contributed by NM,
...
-      LZUKMZBNANZGJZUILZUKMZBNANUMFGCUFUFFGOZUPUTABVAUOUSUKUNURUIPQRURCLZUTULAB
-      VBUSUJUKURCUIPQRUFUKABSUQFTABDEUAUKABFUBUCUDUE $.
-
     $( Membership in a Cartesian product.  (Contributed by NM, 4-Jul-1994.) $)
     elxp $p |- ( A e. ( B X. C ) <-> E. x E. y ( A = <. x , y >. /\
                ( x e. B /\ y e. C ) ) ) $=
       ( cxp wcel cv wa copab cop wceq wex df-xp eleq2i elopab bitri ) CDEFZGCAH
       ZDGBHZEGIZABJZGCSTKLUAIBMAMRUBCABDENOUAABCPQ $.
 
+    $( Membership in a Cartesian product.  One direction of ~ elxp .
```
Comment (wlammen) [inline set.mm:52959]: This proof adds a dependency on ax-sep, ax-nul and ax-pr to the theorem (via opex, elopab (modified)).  Revert it.

**Review by wlammen — CHANGES_REQUESTED** [review]

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm:52959]: @icecream17 Can you in a separate PR prove the forward direction of elopab (label it elopabi, and relabel the current elopabi to, say, elopabes (for "explicit substitution") or elopabim), and then prove elopab and elxpi from it ?   The proof of that new elopabi is basically a subproof of the current elopab and elxpi, so it permits some "factoring".

Can you tell me you plan to do it by replying here, or else I'll do it.

Comment (icecream17) [reply set.mm:52959]: I'll do it

**Review by benjub — APPROVED** [review]

**Review by wlammen — APPROVED** [review]

**Review by avekens — APPROVED** [review]

**Pull request merged**